src/services/dashboards/widgets/cost-by-region/CostByRegion.vue

```diff
@@ -189,16 +295,38 @@ defineExpose<WidgetExpose<FullData>>({
         @apply grid grid-cols-12;
         height: 100%;
         padding-left: 1.5rem;
-        padding-bottom: 2.625rem;
         .chart-loader {
             @apply col-span-5;
             height: 100%;
+            padding-bottom: 1rem;
             .chart {
-                height: 100%;
+                height: 90%;
+            }
+            .legend-wrapper {
+                .circle-wrapper {
+                    display: inline-flex;
+                    align-items: center;
+                    width: 100%;
+                    .circle {
+                        @apply inline-block rounded-full;
+                        margin-right: 0.25rem;
+                        width: 0.5rem;
+                        height: 0.5rem;
+                    }
+                    .label {
+                        @apply mr-4 text-gray-500;
+                        font-size: 0.75rem;
+                        line-height: 1.5;
+                        white-space: nowrap;
```

Comment:
```
.label {
  @apply mr-4 text-label-sm text-gray-700;
  white-space: nowrap;
}
```
How about code like the one above according to the new typo convention (`text-label-sm`)?

Reply:
oh I'll update this later commit!!